pyartcd/pyartcd/pipelines/update_golang.py

```
@@ -396,27 +396,38 @@ async def run(self):
+                        self._rebase_and_build_konflux(el_v, go_version, el_nvr_map_for_images[el_v]),
+                    )
+                    for el_v in sorted(konflux_missing)
                 )
-            if build_tasks:
-                results = await asyncio.gather(*build_tasks, return_exceptions=True)
-            errors = [r for r in results if isinstance(r, Exception)]
-            if errors:
-                error_msgs = "\n".join([str(e) for e in errors])
-                raise RuntimeError(f"{len(errors)} image build(s) failed:\n{error_msgs}")
+            if build_targets:
+                labels, coros = zip(*build_targets)
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)